### DISCUSSION TITLE
ISO8601 date string with T not fetching record of particular date but with space it does.

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2931] [Q&A]
I am able to see records with transactionDate with values 2023-03-31 18:30:00 UTC in pocketbase database but when tried to query as transactionDate >="2023-03-31T18:30:00.000Z" its not responding result but when I tried with transactionDate >="2023-03-31 18:30:00.000Z" gives response as well as when I tried to change date to 30 as transactionDate >="2023-03-30T18:30:00.000Z" also gives response. So T present in date during query is get interrupted as something else it seems... Can you let me know whether I am missing something?

Note: This i figured out in Dart SDK then later in dashboard also same kind of response is there.

**Comment by ganigeorgiev** [comment]
The correct `date` field format should be `Y-m-d H:i:s.uZ`.

We don't support dates with `T` as delimiter at the moment (_there are plans in the future to perform `date` field normalizations to also support different time zones, but for now remains a low priority_).

>  when I tried to change date to 30 as transactionDate >="2023-03-30T18:30:00.000Z" also gives response

It works but the result may not be what you expect because internally the `date` field values are compared as strings (aka. you'll have something like `"2023-03-30 18:30:00.000Z" >= "2023-03-30T18:30:00.000Z"`).
